webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -261,7 +261,7 @@ export default {
       return this.newsId !== null;
     },
     postDisabled: function () {
-      return this.uploading || !this.news.title || !this.news.title.trim() || !this.news.body || !this.news.body.replace(/<p>&nbsp;<\/p>/g, '').replace(/&nbsp;/g, '').trim();
+      return this.uploading || !this.news.title || !this.news.title.trim() || !this.news.body || !this.news.body.replace(/<[^>]*>?/gm, '').replace(/&nbsp;/g, '').trim();
```

Comment:
This regex may remove text even if it has just < charcter without an ending > charcter.
could you use https://developer.mozilla.org/en-US/docs/Web/API/DOMParser ?

Reply:
Okay